app/components/CarouselCard.tsx

```diff
@@ -119,11 +127,21 @@ const $meta: ViewStyle = {
   marginBottom: spacing.medium,
 }
 
+const $mr: TextStyle = {
```

Comment:
Minor nitpick, could we use a more explicit name here? Maybe `$leftButton` to match what it's styling like the others.

Reply:
We could possibly change the `$button` style to `$rightButton` as well.

Reply:
I agree! will update them.

Reply:
Actually that `$button` is just for a regular button not the style for the `rightButton`

Reply:
Look like I can't push anymore after the merge. Will update this in my other PR.